src/classes/AccountDeDuper.cls

```diff
@@ -1,7 +1,7 @@
-public with sharing class AccountDeDuper {
+public with sharing class Account_DeDuper {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.